vpc/service/assign_addresses_v3.go

```diff
@@ -1623,3 +1623,53 @@ func assignSpecificIPv4AddressV3(ctx context.Context, tx *sql.Tx, branchENI *ec2
 		PrefixLength: uint32(prefixlength),
 	}, nil
 }
+
+func assignNextIPv6Address(ctx context.Context, tx *sql.Tx, subnetid string) (net.IP, error) {
+	sequenceName := fmt.Sprintf("v6seq_%s", strings.ReplaceAll(subnetid, "-", "_"))
+	_, err := tx.ExecContext(ctx, fmt.Sprintf("CREATE SEQUENCE IF NOT EXISTS %s MINVALUE 0 MAXVALUE 65534 CACHE 10 CYCLE", sequenceName))
```

Comment:
At first glance this looks wasteful to do on every new IP. 
Is there not some existing procedure for adding new subnets to the vpc service? 
Or is that something we are trying to get away from?


If there was, then this CREATE SEQ could be part of the "add subnet to titus procedure", then we wouldn't have to call it every time we make add an IP?
Then again, I don't know how expensive this is, it is just "one" db call and it does nothing 99% of the time.

Reply:
This is a temporary thing. It's relatively cheap, and takes a shared read lock out most of the time. Honestly, though, this is "low hanging fruit" optimization, and we can do all sorts of tricks, like swapping around the sequence number and putting it first, and using micros for the LSB.